src/composed/key.rs

```diff
@@ -301,7 +314,10 @@ mod tests {
         let input = ::std::str::from_utf8(raw).unwrap();
         let key = PublicKey::from_string(input).expect("failed to parse key");
 
-        // assert_eq!(key.primary_key.fingerprint(), "56c65c513a0d1b9cff532d784c073ae0c8445c0c");
+        assert_eq!(
+            hex::encode(key.primary_key.fingerprint()),
+            "56c65c513a0d1b9cff532d784c073ae0c8445c0c"
```

Comment:
there was already hex_literal used in another test, can you replace that with this as well, no need to have to crated for hex decoding and this one looks nicer

Reply:
Will do :)